Zend/tests/required_param_after_optional.phpt

```diff
@@ -9,6 +9,6 @@ function test3(Type $test3A = null, Type2 $test3B = null, $test3C) {}
 
 ?>
 --EXPECTF--
-Deprecated: Required parameter $testC follows optional parameter $testA in %s on line %d
+Deprecated: Required parameter #2 ($testC) should not follow optional parameter $testA in %s on line %d
```

Comment:
Or `Required parameter #2 ($testC) should precede all optional parameters`? I like positive sentences.